functions/nodejs-apps.bash

```diff
@@ -272,13 +275,16 @@ zigbee2mqtt_setup() {
       if ! (whiptail --title "Zigbee2MQTT installation" --yes-button "Continue" --no-button "Cancel" --yesno "$z2mInstalledText" 14 80); then echo "CANCELED"; return 0; fi
     fi
 
+    if ! cond_redirect fix_permissions /opt/zigbee2mqtt "zigbee2mqtt:${username:-openhabian}" 644 755; then echo "FAILED (zigbee2mqtt set permissions)"; retval=1; fi
+    if ! cond_redirect fix_permissions /var/log/zigbee2mqtt "zigbee2mqtt:${username:-openhabian}" 644 755; then echo "FAILED (zigbee2mqtt set permissions)"; retval=1; fi
+
     echo -n "$(timestamp) [openHABian] Updating Zigbee2MQTT... "
     if ! cond_redirect cd /opt/zigbee2mqtt; then echo "FAILED (cd zigbee2mqtt)"; return 1; fi
     if ! cond_redirect systemctl stop zigbee2mqtt ; then echo "FAILED (stop systemctl)"; fi
-    if ! cond_redirect sudo -u "${username:-openhabian}" cp -R data data-backup; then echo "FAILED (cp backup)"; return 1; fi
-    if ! cond_redirect sudo -u "${username:-openhabian}" git pull; then echo "FAILED git"; return 1; fi
-    if ! cond_redirect sudo -u "${username:-openhabian}" npm ci; then echo "FAILED npm"; return 1; fi
-    if ! cond_redirect sudo -u "${username:-openhabian}" cp -R data-backup/* data; then echo "FAILED (cp backup)"; return 1; fi
+    if ! cond_redirect sudo -u zigbee2mqtt cp -R data data-backup; then echo "FAILED (cp backup)"; return 1; fi
```

Comment:
no sudo
openhabian-config *is* running as root

Reply:
The commands should be executed as zigbee2mqtt not as root. That's what the "sudo -u zigbee2mqtt" does.